Update local account terminology to key-pair (#37)
radicle-dev/registry.radicle.xyz: docs/getting-started.md

### Submitting a transaction

> An account has an address which is determined by the key pair

Reviewer: I think we can elaborate here and more clearly relate it to the goal of submitting a transaction. This would basically move the explanation you do further down here.
```suggestion
To submit a transaction you need an account on the block chain. An account is controlled by a cryptographic key pair. Key pairs are managed by `radicle-registry-cli` and stored on disk. An account is identified by a SS58 address which is derived from the key pair.
An account has an address which is determined by the key pair
```

Author: Sorry, I messed up with that line, didn't mean to include it.

I like your proposal, although I think it overloads the user with very technical information without a practical context to it, or not even, I think.

For example:

> To submit a transaction you need an account on the block chain. 

As a user, I would ask _why_? The only answer I can think of is so to pay for tx fees, therefore we could better be more transparent in that respect.

Reviewer: > As a user, I would ask _why_? The only answer I can think of is so to pay for tx fees, therefore we could better be more transparent in that respect.

Good point. I think it would make sense to add this.

> I like your proposal, although I think it overloads the user with very technical information without a practical context to it, or not even, I think.

I don’t think this is information overload—on the contrary. All the terms introduced and put into relation here are used further down without context. For example the sentence “You can always obtain the account addresses for all your key pair” is the first to mention an account address but fails to explain its relation to the a key pair.

In addition the sentence here explains why we need to generate a key pair. Something that is hinted at in the sentence “You need a key pair to author transactions” below.

Author: > I don’t think this is information overload

I wasn't clear: the information is good, I just missed the explanation as to why some bits were so; under that light I found the info to be overloading.

Anyway, I have reworked this part to integrate your latest suggestion, which I have slightly touched. Let me know how you like it now.

> 1. Generate a key pair
> 
>     You need a key pair to author transactions.

Reviewer 2: We didn't get rid of the `account` notion completely, it's still used below. I propose to keep it properly introduced.
```suggestion
1. Generate an account and its key pair

    You need an account to author transactions.
    An account doesn't need to be explicitly created, as soon as you create a key pair,
    you can start using a new account controlled with it.
```

Author: I think this is properly addressed in this running discussion: https://github.com/radicle-dev/registry.radicle.xyz/pull/37#discussion_r419305153

    The key pairs are randomly generated and identified by a name of your choice.
    To generate a new key pair, run:

    ```bash
    radicle-registry-cli key-pair generate <name>
    ```

    This command will give you the SS58 address for your new key pair.
    E.g. generating a key pair named 'neo':

    ```bash
    radicle-registry-cli key-pair generate neo
    ✓ Key pair generated successfully
    ⓘ SS58 address: 5HWP48i9TuP2VrRZrNeb6QzYpdUSE9BvyaptEqqHrLfH8ZPd
    ```

>     You can always obtain the addresses of all your key pairs by running:

Reviewer: ```suggestion
    You can always obtain the account addresses for all your key pairs by running:
```

### Submitting a transaction

> 1. Generate a key-pair

Reviewer: I think this section would benefit from more clearly explaining the relation ship between accounts and key pairs stored locally. Specifically we should make it clear that an account lives on chain and is controlled by a key pair. An account has an address which is determined by the key pair. I’d suggest saying “account address for a key pair” instead of “key pair address” to make this more obvious.

Author: > I think this section would benefit from more clearly explaining the relation ship between accounts and key pairs stored locally

Yes, absolutely.

It actually would make more sense to have it in the next sub-section, `Request RADs`, where this understanding becomes important, don't you think? Will commit a suggestion, can you have a look?

    You need a key-pair to author transactions.
    The key-pairs are randomly generated and identified by a name of your choice.
    To generate a new key-pair, run:

    ```bash
    radicle-registry-cli key-pair generate <name>
    ```

    This command will give you the SS58 address for your new key-pair.
    E.g. generating a key-pair named 'neo':

    ```bash
    radicle-registry-cli key-pair generate neo
    ✓ Key-pair generated successfully
    ⓘ SS58 address: 5HWP48i9TuP2VrRZrNeb6QzYpdUSE9BvyaptEqqHrLfH8ZPd
    ```

    You can always obtain the addresses of all your key-pairs by running:

    ``` bash
    radicle-registry-cli key-pair list
    ```

2. Request RADs

    To submit transactions, you first need to get some RAD into your key-pair

>     account to have enough funds to pay for transactions fees.

Reviewer: This introduces the unexplained term “key pair account”. I think just “account” should suffice.

    Get in touch with us at <a href="https://webchat.freenode.net/#radicle" target="_blank" rel="noopener noreferrer">irc://irc.freenode.net/#radicle</a>

>     to request some RAD for your key-pair's account. Please, bring your SS58

Reviewer: As abov, just “account” should suffice.
